src/copilot/app/rag_api.py

```diff
@@ -30,23 +39,26 @@
 )
 
 
-@app.post("/rag/process", summary="Process RAG query endpoint", response_description="Return result from processing RAG query", status_code=200)
+@app.post("/query",
```

Comment:
rename api route to "/process" as copilot-frontend currently makes calls to "/rag/process" route.

Reply:
I changed in the route call in the frontend to match /query